src/board.py

```diff
@@ -127,6 +127,9 @@
 elif board_id == ap_board.PINE64:
     from adafruit_blinka.board.pine64 import *
 
+elif board_id == ap_board.ONION_OMEGA:
```

Comment:
Can you make sure you're updating the correct branch? I still see Onion Omega stuff here.

Reply:
hahaha my fault, I did not push the changes.